examples/data-objects/codemirror/src/presence.ts

```diff
@@ -23,21 +23,21 @@ interface IColor {
     };
 }
 
+const presenceKey = `PresenceManager-6eb56a01-af1a-4b94-9d36-c62ec587caa6`;
```

Comment:
this shouldn't need the guid. signals submitted from the datastore runtime already have the datastore id encoded in the envelope, so unqiueness shouldn't be a concernt
```suggestion
const presenceKey = `PresenceManager`;
```

Reply:
The reason I've added guid to underlying that the name itself (PresenceManager) unique enough and there might be other "presence managers" used on same runtime. This reduces assumptions and leaves only one - there should be only one instance of this class used per data store runtime, no other restrictions.

I'm happy to add comment or remove - it's not such a big deal either way.